.husky/prepare-commit-msg

```diff
@@ -0,0 +1,4 @@
+[ -n "$CI" ] && exit 0
+
+exec < /dev/tty
+node ./lib/cli.js --hook || true
```

Comment:
I think it should be this:

```bash
node ./bin/git-cz --hook || true
```

Granted that means that would only work after `yarn build`

In lieu of this change, should 100% update README on how husky uniquely functions where in an import one should be calling:

```bash
node ./node_modules/.bin/gitcz --hook || true
```

As there seems to be a clash with `git-cz`. This took an inordinate amount of time to figure out, so could also be user error on my part.

Husky does recommend calling `yarn` directly so perhaps but I cannot get that to work for the life of me.